includes/object-cache.php

```diff
@@ -1056,11 +1058,16 @@ public function fetch_info() {
         }
 
         if ( defined( 'WP_REDIS_CLUSTER' ) ) {
-            $connectionID = current( $this->build_cluster_connection_array() );
+            $client = $this->determine_client();
+	        $connectionID = current( $this->build_cluster_connection_array() );
 
-            $info = $this->determine_client() === 'predis'
-                ? $this->redis->getClientFor( $connectionID )->info()
-                : $this->redis->info( $connectionID );
+	        if ( $client === 'predis' ) {
+                $info = $this->redis->getClientFor( $connectionID )->info();
+            } elseif ( $client === 'phpredis' ) {
+                $info = $this->redis->info( 'SERVER' );
```

Comment:
@alexva24: Final question, I see that the only big change here is that we're calling `RedisCluster::info( 'SERVER' );` instead of `RedisCluster::info();`. What exact error message is this fixing?

Reply:
The PhpRedis syntax is:

```php
public function info(string|array $node, ?string $section = null): array;
```

So I don't think this works?

Reply:
**The method RedisCluster::info() signature is:**
* @param   string $option Optional. The option to provide redis.
*                         SERVER | CLIENTS | MEMORY | PERSISTENCE | STATS | REPLICATION | CPU | CLASTER | KEYSPACE  | COMANDSTATS
    _public function info($option = null) {}_

Using info( 'SERVER' ) reduced count of returning data. See [https://redis.io/commands/info/#notes](https://redis.io/commands/info/#notes)

I tested my changes and already use it my production server.

Reply:
Error message"Warning RedisCluster::info(): Directed commands must be passed a key …
…or [host,port] array"

Reply:
https://github.com/phpredis/phpredis/blob/677c9da3cbff53b74f7b26ca7dcbdf7e37e1da04/redis_cluster.stub.php#L165

That's the correct behavior.

Reply:
So, i made some changes